docs/fastdds/transport/transport_api.rst

```diff
@@ -169,3 +190,87 @@ However, :class:`IPLocator` allows to manage them if needed.
     :end-before: //!--
     :dedent: 8
 
+
+Chaining of transports
+----------------------
+
+There are use cases where the user needs to pre-process out-coming information before being sent to network and also
+the incoming information after being received.
+*Transport API* offers two interfaces for implementing this kind of functionality: :class:`ChainingTransportDescriptor`
+and :class:`ChainingTransport`.
```

Comment:
These classes are not included in the API reference, though neither are the other `TransportDescriptor`s. Being part of the public API, they should be added. It could be done in another PR but it would be nice to have a task open in Redmine so this issue can be tracked.

Reply:
Created task 13193